pam/internal/adapter/newpasswordmodel.go

```diff
@@ -72,26 +73,53 @@ func (m newPasswordModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 
 	case tea.KeyMsg: // Key presses
 		switch msg.String() {
-		case "enter":
-			if m.focusIndex >= len(m.focusableModels) {
+		case "tab":
```

Comment:
Since you did the logic already, can you also support `shift+tab` by inverting the calls on `focusPrev/Next`?